template.yaml

```diff
@@ -107,6 +107,20 @@ Resources:
             RestApiId: !Ref GatewayAPI
             Path: /url/{id}
             Method: GET
+  
+  TelegramWebhookFunction:
```

Comment:
@alexvingg I will leave you with this sample so you can work on it, sounds good?